### ISSUE TITLE
Snap! doesn't play sounds on mobile devices

### ISSUE THREAD

**Issue by ddgarcia** [jmoenig/Snap#235]
http://tinyurl.com/bjcmolemash

...plays wonderful sounds on all computers but on no mobile devices. Can this be fixed for the Hour of Code asap?

Thanks!

dan

**Comment by cycomachead** [comment]
Updating this after a long while:
There might be a solution if we could somehow initialize the sounds on each tap event. (Like a hat block / green flag.)

https://twitter.com/cycomachead/status/758212145284730881

**Comment by sinuhe69** [comment]
Yes, I can confirm that the issue still exists on mobile devices. I tested on 2 Android devices (Kitkat 4.2.1 and 4.4), the "Play Sound" block doesn't work in Chrome or Firefox for Mobile (both latest versions). However, when I click on the Play button, I can hear the sound without any problem! It's really weird :(

Demo here: https://goo.gl/rZlO2O
